v2/testdata/ocspidpkixocspnocheckextnotincluded_SMIME_CA_correct.pem

```diff
@@ -0,0 +1,50 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I notice the test files also lack their "openssl x509" output. You can use https://github.com/zmap/zlint/blob/aa4e2619db00dbdea768e9afa70262f0d6af3417/v2/test/prepend_testcerts_openssl.sh to generate that output for the .pem files.

Reply:
@sleevi where do you want the "openssl x509" output to be stored? Do you have an example from a different test?

Reply:
@RufusJWB You can run the `prepend_testcerts_openssl.sh` script and it will literally add the output to the front of these `.pem` files where it will be ignored by the `x509` package at parse-time but will be present for human inspection.

Reply:
Also I thought I landed a CI task that failed if there were `.pem` files missing this prepended output. Sounds like it might not be working correctly :-(

Reply:
@sleevi I used this handy der2ascii tool to reduce the size of the certificates significantly. Obviously the signature is now broken, but I think this is okay for this use case, isn't it?

Reply:
@cpu It seems that something is wrong with `prepend_testcerts_openssl.sh`

> z002m76a@md1r0uxc:/mnt/c/Repositories/zlint/v2/test$ ./prepend_testcerts_openssl.sh
> ./prepend_testcerts_openssl.sh: line 17: warning: command substitution: ignored null byte in input
> ./prepend_testcerts_openssl.sh: line 17: warning: command substitution: ignored null byte in input
> ./prepend_testcerts_openssl.sh: line 17: warning: command substitution: ignored null byte in input
> z002m76a@md1r0uxc:/mnt/c/Repositories/zlint/v2/test$   

any thoughts on this?